I see that the GHA checks failed on the [Compare two databases](https://github.com/dtcenter/METdataio/actions/runs/3290789052/jobs/5424097071) job. But I do think the error makes sense:
```
pymysql.err.ProgrammingError: (1146, "Table 'mv_ci_prod.line_data_seeps' doesn't exist")
Error: Process completed with exit code 1.
```
The *old* database doesn't contain a table for the new SEEPS line type but the *new* table does since that's what @venitahagerty added. The new database should also contain a new table for SEEPS_MPR, but I'm guessing the comparison is erroring out before getting to that one.

@TatianaBurek and @venitahagerty, you could consider making the database comparison logic more robust to handle differences in which tables actually exist. But I certainly don't think that this error indicates that there's anything wrong with this PR.

But I defer to @TatianaBurek in giving her final review approval on this PR.

--- a/METdbLoad/sql/mv_mysql.sql
+++ b/METdbLoad/sql/mv_mysql.sql
@@ -449,7 +449,7 @@ CREATE TABLE line_data_ecnt
     crps_emp         DOUBLE,
     crpscl_emp       DOUBLE,
     crpss_emp        DOUBLE,
-    crpss_emp_fair   DOUBLE,
+    crpss_emp_fair   DOUBLE DEFAULT -9999,
 
     CONSTRAINT line_data_ecnt_data_file_id_pk
         FOREIGN KEY (data_file_id)

